### PR TITLE
Point to version of cartodb.js that includes mustache.js v1.1.0

### PR DESCRIPTION
**Pull request by alonsogarciapablo**

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2359/))

**Comment by alonsogarciapablo** [comment]
This is related to #2420.

@javisantana @xavijam can you please take a look?

File: vendor/assets/javascripts/cartodb.uncompressed.js [hunk 1]
```diff
@@ -21355,7 +21408,15 @@ cdb.core.Template = Backbone.Model.extend({
...
+    'mustache': typeof(Mustache) === 'undefined' ?
+      null :
+      // Replacement for Mustache.compile, which was removed in version 0.8.0
+      function compile(template) {
+        Mustache.parse(template);
+        return function (view, partials) {
+          return Mustache.render(template, view, partials);
+        };
+      }
   },
```
Comment (xavijam) [inline vendor/assets/javascripts/cartodb.uncompressed.js]: :+1:

**Comment by xavijam** [comment]
Looks :+1: for me.

**Comment by alonsogarciapablo** [comment]
@javisantana should I merge this guy?

**Comment by javisantana** [comment]
yes (ci is failing tho)

**Comment by xavijam** [comment]
retest this please

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2402/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2403/))

**Comment by alonsogarciapablo** [comment]
I'm going to point to cartodb.js v.3.12.13.

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2417/))

**Pull request merged**